### DISCUSSION TITLE
is it possible to reference current row in expressions as Dict?

### DISCUSSION THREAD

**Discussion by jan-kaczorowski-mx** [saulpw/visidata#2539] [Q&A] [answered]
when I use python expressions to create a new column after clicking "=", is there any way to reference current row as Dict rather than list? I'd find it helpful in cases when I have columns like `"Silly Named Column With Spaces"`. I know I can refer it like `row[this_column_idx]` but that's not very handy as you have to be able to tell the index of particular column. What I can't do is `row["Silly Named Column With Spaces"]`. Something like `row_as_dict["Silly Named Column With Spaces"] would be excellent, but maybe there already is a solution to it I'm not aware of?

**Comment by saulpw** [comment, marked as answer]
I think you're right, that `row` should work like this.  I've renamed the old row to `_row` and now `row["Silly Named Column With Spaces"]` should work in the develop branch and is slated for v3.1 soon.
